src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,85 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { createComment } from '../api/api';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  setComments: (arg: Comment[]) => void,
+  POST_ID: number,
+  setCommentError: (arg: boolean) => void,
+  comments: Comment[] | null,
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  setComments, POST_ID, setCommentError, comments,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
```

Comment:
You can combine state into one instead of 3
```
useState({
  name: '',
  ...
})
```
After than you will need one onChange handler
```
const onChange = (e) => {
  const {name, value} = e.target;
  
  setState(prev => ({...prev, [name]: value}))
}
```